pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about-assemblies.adoc

```diff
@@ -8,7 +8,7 @@
 = About Assemblies
 
 [role="_abstract"]
-In {ProductName} {ProdctNumber}, each assembly encompasses a single user story. An assembly can include concept, procedure, or reference modules but all modules in an assembly must relate to that specific user story. Previously, many Red Hat documentation teams used assemblies as the structure of user guides and other "books." In {ProductName} {ProductNumber}, there are no books, only user stories, with one assemply per user story. This change is part of our efforts to make it easier for users to search for each specific action they want to perform. 
+In {ProductName} {ProdctNumber}, each assembly encompasses a single user story. An assembly can include concept, procedure, or reference modules but all modules in an assembly must relate to that specific user story. Previously, many Red Hat documentation teams used assemblies as the structure of user guides and other "books." In {ProductName} {ProductNumber}, there are no books, only user stories, with one assembly per user story. This change is part of our efforts to make it easier for users to search for each specific action they want to perform. 
 
 Link:https://redhat-documentation.github.io/modular-docs/#forming-assemblies[Assemblies] are discussed in detail in our company's description of modular documentation. Here are specific guidelines for creating assemblies in {ProductName} {ProductNumber}:
```

Comment:
I suggest putting this under .Additional resources, which is compatible with the mod docs guidelines.